packages/wdio-cli/src/commands/config.ts

```diff
@@ -192,7 +192,11 @@ const runConfig = async function (useYarn: boolean, yes: boolean, exit = false)
 }
 
 export function handler(argv: ConfigCommandArguments) {
-    return runConfig(argv.yarn, argv.yes)
+    if (argv.framework && !SUPPORTED_PACKAGES.framework.find(({ name }) => name === argv.framework)) {
+        throw new Error(`InvalidArgumentError: Framework ${argv.framework} is not supported.`)
+    }
+
+    return runConfig(argv.yarn, argv.yes, argv.framework)
```

Comment:
Given that we essentially want to allow to provide answers or all questions I suggest to pass in an object rather than defining individual answers. I believe we can re-use the `QUESTIONNAIRE` and and loop over this.